client/api/eventsApi.js

```diff
@@ -2,6 +2,13 @@ import { getEncodedToken } from 'authenticare/client'
 import request from 'superagent'
 const acceptJsonHeader = { Accept: 'application/json' }
 
+/**
+ * Post new event to projects table
+ * @param {string} id - User Id
+ * @param {object} event - Form field inputs from CreateProject
+ *
+ * @returns {array} array[0] eventId
```

Comment:
```suggestion
 * @returns {Promise.<number[]>} array[0] eventId
```
💭 Technically, this function returns a promise, not an array.
💭 Not documented in jsdocs website, but some IDEs understand that `Promise.<something>` documents the type of value that the promise resolves with.